src/librustc_save_analysis/lib.rs

```diff
@@ -210,7 +210,7 @@ impl<'l, 'tcx: 'l> SaveContext<'l, 'tcx> {
                     decl_id: None,
                     docs: self.docs_for_attrs(&item.attrs),
                     sig: sig::item_signature(item, self),
-                    attributes: lower_attributes(item.attrs.clone(), self),
+                    attributes: lower_attributes(item.attrs.to_owned(), self),
```

Comment:
seems weird to go from `clone` to `to_owned`. One wouldn't call `to_owned` on a `&String`, only on a `&str` I think?

Reply:
Both variants are currently present in the file and I thought that `to_owned()` would be preferable here, but I don't mind performing the change the other way around.